### ISSUE TITLE
Donations

### ISSUE THREAD

**Issue by Metaception** [evilsocket/opensnitch#215]
Is there anyway to donate to this project?

Is it too much trouble and/or you would rather have packagers/contributors?

**Comment by Unboncafe** [comment]
Hi ! 
Yes, it's really a good idea to you and the other contributors to have some money 👍 
Bitcoin adress, Paypal option, Neosurf (payment in France, anonym) ...

**Comment by Pain-Patate** [comment]
Have you seen how we can to donate to help you

**Comment by gustavo-iniguez-goya** [comment]
Yes, @Pain-Patate / @Metaception  , finally I've setup my github's sponsors page: 
https://github.com/sponsors/gustavo-iniguez-goya
And also liberapay: https://liberapay.com/gus/

**Comment by jermanuts** [comment]
@gustavo-iniguez-goya do you plan on accepting cryptocurrencies? Monero would be great choice.

**Comment by gustavo-iniguez-goya** [comment]
hey @jermanuts , thank you very much for your interes, check out the sponsors page: https://github.com/sponsors/gustavo-iniguez-goya
